src/scenes/ProgressReports/Detail/WorkflowEventsHistory.tsx

```diff
@@ -0,0 +1,76 @@
+import { East } from '@mui/icons-material';
+import { Box, Collapse, Divider, Skeleton, Typography } from '@mui/material';
+import { Fragment } from 'react';
+import { ProgressReportStatusLabels as StatusLabels } from '~/api/schema/enumLists';
+import { FormattedDate } from '~/components/Formatters';
+import { Redacted } from '~/components/Redacted';
+import { RichTextView } from '~/components/RichText';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+
+interface WorkFlowEventHistoryProps {
+  report?: ProgressReportDetailFragment;
+  checked?: boolean;
+}
+
+export const WorkFlowEventHistory = ({
+  checked,
+  report,
+}: WorkFlowEventHistoryProps) => {
+  if (!report?.workflowEvents) {
+    return <Skeleton variant="rectangular" width="55%" />;
+  } else if (report.workflowEvents.length === 0) {
+    return (
+      <Redacted
+        info="You do not have permission to view this information"
+        width="60ch"
+      />
+    );
+  }
+
+  const statusEvents = [
+    {
+      ...report.workflowEvents[0]!,
+      status: 'NotStarted' as keyof typeof StatusLabels,
+    },
+    ...report.workflowEvents,
+  ];
+
+  return (
+    <>
+      {statusEvents.slice(0, statusEvents.length - 1).map((event, i) => (
+        <Fragment key={i}>
+          <Box
+            sx={{
+              py: 2,
+              px: 3,
+              bgcolor: 'primary.contrastText',
+              borderRadius: 1,
+            }}
+          >
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Typography variant="body1" sx={{ fontWeight: 'bold' }}>
+                {StatusLabels[event.status]}
+              </Typography>
+              &nbsp;
+              <East />
+              &nbsp;
```

Comment:
```suggestion
              <ArrowForward aria-label="transitioned to" sx={{ mx: 1 }} />
```
Arrow forward more represents our intent here instead of Map East. The icons are only slightly different so this is mostly just on principle.
Using `aria-label` and margin is better for screen readers.